### PR TITLE
DTGB-499: Fixed styling of icons in gentinfo block.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/31-molecules/islands/gentinfo-island/gentinfo-island.twig [hunk 1]
```diff
@@ -11,11 +11,11 @@
...
     {% include '@list'
       with {
-        'type': 'unordered',
-        'items': [
-          '<a href="tel:09 210 10 10" alt="Gentinfo tel link" class="button button-secondary button-call icon-phone">T: 09 210 10 10</a>',
-          '<a href="mailto:gentinfo@stad.gent" alt="Gentinfo mail link">Gentinfo@stad.gent</a>',
-          '<a href="javascript:(void);" class="standalone-link" alt="Contact form">Contactformulier</a>'
+        'type': 'icon-list',
+        'icon_items': [
+          '<i class="icon-phone" aria-hidden="true"></i><a href="tel:09 210 10 10" alt="Gentinfo tel link">09 210 10 10</a>',
```
Comment (Jeroen005) [inline styleguide/components/31-molecules/islands/gentinfo-island/gentinfo-island.twig]: trek het alvast door op de Drupal theming :) href hoort in internationaal format te zijn

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: styleguide/components/31-molecules/islands/gentinfo-island/gentinfo-island.twig [hunk 2]
```diff
@@ -13,7 +13,7 @@
       with {
         'type': 'icon-list',
         'icon_items': [
-          '<i class="icon-phone" aria-hidden="true"></i><a href="tel:09 210 10 10" alt="Gentinfo tel link">09 210 10 10</a>',
+          '<i class="icon-phone" aria-hidden="true"></i><a href="tel:+32 210 10 10" alt="Gentinfo tel link">09 210 10 10</a>',
```
Comment (Jeroen005) [inline styleguide/components/31-molecules/islands/gentinfo-island/gentinfo-island.twig]: @meirege serieus 🤦‍♂️ , het zonenr he.

File: styleguide/components/41-organisms/document-box/_document-box.scss [hunk 3]
```diff
@@ -1,14 +1,6 @@
 .document-box {
-  @include theme('border-color', 'color-primary--lighten-4', 'more-info-block-border-color');
-
   position: relative;
   margin-bottom: 2rem;
```
Comment (Jeroen005) [inline styleguide/components/41-organisms/document-box/_document-box.scss]: margin-bottom mag hier toch ook weg? wrm position relative nodig? vanwege het icon, dan staat het hier verkeerd :)

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: styleguide/components/41-organisms/document-box/_document-box.scss (hunk 3 above)
Comment (meirege) [reply styleguide/components/41-organisms/document-box/_document-box.scss]: Jup.

File: styleguide/components/31-molecules/islands/gentinfo-island/gentinfo-island.twig (hunk 2 above)
Comment (meirege) [reply styleguide/components/31-molecules/islands/gentinfo-island/gentinfo-island.twig]: Idd :( ...

**Review by Jeroen005 — APPROVED** [review]

**Pull request merged**